.travis.yml

```diff
@@ -6,10 +6,20 @@ rvm:
 - "1.9"
 - "2.0"
 - "2.1"
-- "2.2"
+- "2.2.2"
 
 gemfile:
 - Gemfile.activesupport32
 - Gemfile.activesupport40
 - Gemfile.activesupport41
 - Gemfile.activesupport42
+- Gemfile.activesupport50
+
+matrix:
+  exclude:
+    - rvm: "1.9"
```

Comment:
2.0 is also no longer supported, but we may want to keep it because it's the Ruby version OSX ships with.